src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenExpression.vb

```diff
@@ -3214,9 +3214,9 @@ Class C1
         CheckDouble(Integer.MaxValue + 0D, Integer.MaxValue)
         CheckDouble(Integer.MaxValue + 0.01D, Integer.MaxValue)
         ' https://github.com/dotnet/roslyn/issues/74026
-        ' CheckDouble(Integer.MaxValue + 0.99D, Integer.MinValue)  ' overflow
-        ' CheckDouble(Integer.MaxValue + 1D, Integer.MinValue)  ' overflow
-        ' CheckDouble(Integer.MaxValue + 2D, Integer.MinValue)  ' overflow
+        CheckDouble(Integer.MaxValue + 0.99D, ]]><%= If(ExecutionConditionUtil.IsMonoOrCoreClr, "Integer.MaxValue", "Integer.MinValue") %><![CDATA[)  ' overflow
+        CheckDouble(Integer.MaxValue + 1D, ]]><%= If(ExecutionConditionUtil.IsMonoOrCoreClr, "Integer.MaxValue", "Integer.MinValue") %><![CDATA[)  ' overflow
+        CheckDouble(Integer.MaxValue + 2D, ]]><%= If(ExecutionConditionUtil.IsMonoOrCoreClr, "Integer.MaxValue", "Integer.MinValue") %><![CDATA[)  ' overflow
```

Comment:
Did you consider changing `CheckDouble` so that it has an overload that takes two arguments: expected framework and expected core? That may make the callsite code a bit cleaner.

Reply:
That could be done but at the price of having to do many more code changes - the CheckDouble/CheckSingle methods are defined in each test separately (so either we would have to change the tests to share them or update each of these tests), I'm not sure that's worth it.